.travis.yml

```diff
@@ -1,26 +1,45 @@
-sudo: enabled
```

Comment:
This commit looks like it should be split into at least three separate ones - one to update to xenial, one to apply whitespace changes and one to add the deploy feature